deploy/local/dockerfiles/client.Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:20.12.2
+
+WORKDIR /jepret-client
+COPY ./package.json ./package-lock.json ./
+RUN npm install
+
+COPY ./client/ ./
+
+ENTRYPOINT ["npm", "run", "dev"]
```

Comment:
Yes! Using `npm run dev` is suited to our context here as local development. 🎉 Just quick note for the production environment better use `npm run build`